lib/Parse/ParseDecl.cpp

```diff
@@ -1105,11 +1123,21 @@ Parser::parseDifferentiatingAttribute(SourceLoc atLoc, SourceLoc loc) {
           /*afterDot*/ false, original.Loc,
           diag::attr_differentiating_expected_original_name,
           /*allowOperators*/ true, /*allowZeroArgCompoundNames*/ true);
+      
+      if (consumeIfTrailingComma())
+        return makeParserError();
     }
-
-    // Parse the optional comma and `wrt` differentiation parameters clause.
-    if (consumeIf(tok::comma) &&
-        Tok.is(tok::identifier) && Tok.getText() == "wrt" &&
+    
+    // Parse the optional `linear` differentiation flag.
```

Comment:
Nit: please use single quotes for literal text.
```suggestion
    // Parse the optional 'linear' differentiation flag.
```

Reply:
Ahh yes my mistake I did notice it used in other places, just a habit of mine to use the grave accent .